src/toast/map/pixels.py

```diff
@@ -16,6 +16,8 @@
 
 from ..cache import Cache
 
+from .._libtoast import global_to_local as libtoast_g2l
```

Comment:
Renaming `global_to_local` as `g2l` seems less than optimal. Anyone seeing that will be forced to scroll to the start of the file to figure out what `g2l` does. Can we either keep `global_to_local` or call it `libtoast_global_to_local`?

Reply:
I was avoiding importing this as "global_to_local" since there is already a class method by the same name.  Trying to distinguish between the function from the bindings and the DistPixels member function.  As long as they are different, I don't care what name is used for the imported function.